test/acceptance/behave/README.md

```diff
@@ -62,15 +66,19 @@ Note: We recommend the use of virtualenv, because is an isolated working copy of
 ### Tests execution:
 
 - Change to the test/acceptance/behave folder of the project if not already on it.
-- You may need to set `export GIT_SSL_NO_VERIFY=true` environment variable in your machine
-- You should add "--upgrade" (if you have a previous installed version of this library)
-- We recommend to create `settings` folder in  behave root directory if it does not exists and store all configurations to referenced by `properties.json` files.
+- We recommend to create `settings` folder in  behave root directory if it does not exists and store all configurations to `properties.json` and `configuration.json` files.
```

Comment:
Disussed by skype, configuration.json should stay in the root directory. Documentation should be clear on this, as with the current working it seems that it has be be placed insid settings/.